src/core/client/ui/components/v2/Flex/Flex.tsx

```diff
@@ -105,7 +105,7 @@ const Flex: FunctionComponent<Props> = (props) => {
   return (
     <Box
       ref={forwardRef}
-      className={cn(rootClassNames, flexClassNames)}
+      className={cn(rootClassNames, flexClassNames, "hahahaha")}
```

Comment:
Looks like this snuck in...can you remove :)?

Reply:
✅